src/AzureADB2C/Provider.php

```diff
@@ -198,11 +204,18 @@ private function validateIdToken($idToken)
      */
     protected function mapUserToObject(array $user)
     {
+        //check is $user['email'] is array - it shouldn't be, but we are not sure
+        if (is_array($user['email'])) {
```

Comment:
I guess we should also make sure this is not empty? Or alt, just do
`$user['email'][0] ?? $user['email']`

Reply:
No worries.